### ISSUE TITLE
Support the REPOSITORY_ID indexing strategy for the demo site

### ISSUE THREAD

**Issue by drh-stanford** [sul-dlss/arclight#269]
Depends on #268. Use a directory name as the REPOSITORY_ID, for example:

* data/
  * ead/
    * repo_a_slug/
      * 1.xml
      * 2.xml
    * a.xml
    * b.xml

Then you'd have an indexing commands like so:

```
REPOSITORY_ID=repo_a_slug bundle exec rake arclight:index_dir DIR=data/ead/repo_a_slug
bundle exec rake arclight:index FILE=data/ead/a.xml
bundle exec rake arclight:index FILE=data/ead/b.xml
```

**Comment by anarchivist** [comment]
This should also be closed by sul-dlss/arclight-demo#17 - we just need to update ArcLight and run a deploy.

**Comment by anarchivist** [comment]
Closed by sul-dlss/arclight-demo#17 and sul-dlss/arclight-demo#18.
